src/main/groovy/com/lesfurets/jenkins/unit/declarative/GenericPipelineDeclaration.groovy

```diff
@@ -14,11 +14,22 @@ abstract class GenericPipelineDeclaration {
 
     static <T> T createComponent(Class<T> componentType,
                                  @DelegatesTo(strategy = DELEGATE_ONLY) Closure<T> closure) {
-        def componentInstance = componentType.newInstance()
+        final def componentInstance = componentType.newInstance()
```

Comment:
Could you please add few comments about `final`. It's might be not so obvious. (I'm also curious)

Reply:
specifying the variable as **final** is a hangover from my java programming days. I added the following comment
`declare componentInstance as final to prevent any multithreaded issues, since it is used inside closure `